google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -73,7 +73,7 @@
   private static final String MANIFEST_ARTIFACT_ID_KEY = "artifactId";
   private static final String MANIFEST_VERSION_KEY = "Implementation-Version";
   private static final String ARTIFACT_ID = "google-cloud-core";
-  private static final String LIBRARY_NAME = "gcloud-java";
```

Comment:
The only Veneer+Vtk client right now is Logging. It doesn't set any headers on the GAPIC client:

https://github.com/GoogleCloudPlatform/google-cloud-java/blob/master/google-cloud-logging/src/main/java/com/google/cloud/logging/spi/DefaultLoggingRpc.java

@bjwatson Can you comment on the user-agent situation? We've been sending `gcloud-java` in user-agent for quite a while now, and I don't know if there are things that depend on it. We avoided changing it to `google-cloud-java` even though everything else was renamed several months ago.